### PR TITLE
fix: add back sticky header & scrollable sidebar

### PR DESCRIPTION
**Pull request by MitchellNZ**
- Removed comment that was blocking sticky/fixed header from working
- Added styles for scrollable sidebar

### CODE REVIEW

**Comment by stsrki** [comment]
Just a simple solution :) Thanks you!

This also fixes #569

File: Source/Blazorise/Styles/_layout.scss [hunk 1]
```diff
@@ -31,7 +31,7 @@
     color: rgba(0, 0, 0, 0.65);
     /*background: #001529;*/
     /*line-height: 64px;*/
-    /*&-fixed {
+    &-fixed {
```
Comment (MitchellNZ) [inline Source/Blazorise/Styles/_layout.scss:34]: @stsrki this is the change I am referring to that I added back.
It looked like you mistakenly commented this line when you were removing `line-height`.

Without this, there was no more `.b-layout-header-fixed` generated in the CSS, which broke the `fixed` header.

Comment (stsrki) [reply Source/Blazorise/Styles/_layout.scss:34]: Yes you're right, sorry about that. Since it's now back I can proceed with merge.

Comment (MitchellNZ) [reply Source/Blazorise/Styles/_layout.scss:34]: Great! Sounds good to me.
Thanks again.

**Pull request merged**